opentelemetry-sdk/src/opentelemetry/sdk/_logs/_internal/__init__.py

```diff
@@ -448,9 +449,6 @@ def __init__(
     ) -> None:
         super().__init__(level=level)
         self._logger_provider = logger_provider or get_logger_provider()
-        self._logger = get_logger(
-            __name__, logger_provider=self._logger_provider
-        )
```

Comment:
This seems like a very significant change. But I am also confused about why the logging handler was storing a logger for it's own source namespace and not even the namespace of the logger it's added to. Can anyone explain the purpose of this?

Reply:
To be more clear. This seems like a _good_ change. I'm just surprised by how strange the existing code's functionality is. So, I wanted to make sure there was not a good reason for it to be that way.

Reply:
It seems a single logger instance stored in `_logger` was chosen based on perceived need for performance, based on the discussion in the original issue https://github.com/open-telemetry/opentelemetry-python/issues/2485 

Based on the discussion yesterday, I'm planning to use pytest-benchmark to test if there is a performance regression with creating one logger instance per logger name.

Reply:
@tm0nk 

Are you planning on including these tests as part of this pr?